zero/spitzer_coll_freq.c

```diff
@@ -9,6 +9,37 @@
 #include <gkyl_spitzer_coll_freq_priv.h>
 #include <gkyl_range.h>
 
+// Calculate the plasma frequency
+double plasma_frequency(double n, double m, double eps0, double hbar, double eV)
+{
+  return sqrt(n*eV*eV/m/eps0);
+}
+
+// Calculate the Coulomb Logarithm
+double coulomb_log(double ns, double nr, double ms, double mr, double Ts, double Tr, double qs, double qr, double bmag_mid, double eps0, double hbar, double eV)
+{
+
+  double vts = sqrt(Ts/ms);
+  double vtr = sqrt(Tr/mr);
+  double wps = plasma_frequency(ns,ms, eps0, hbar, eV);
+  double wpr = plasma_frequency(nr,mr, eps0, hbar, eV);
+  double wcs = qs*bmag_mid/ms;
+  double wcr = qr*bmag_mid/mr;
+  double inner1 = (wps*wps + wcs*wcs)/(Ts/ms + 3*Ts/ms) + (wpr*wpr + wcr*wcr)/(Tr/mr + 3*Ts/ms);
```

Comment:
@akashukla why is this called inner again? I think it has a physical reason. Please comment in code for clarity/interpretability

Reply:
I don't know if there is a physical meaning. To me they were just temporary variable names for the terms that appear inside the logarithm.